lib/tls.js

```diff
@@ -221,12 +221,12 @@ exports.checkServerIdentity = function checkServerIdentity(hostname, cert) {
         valid = wildcard(cn);
 
       if (!valid)
-        reason = `Host: ${hostname}. is not cert's CN: ${cn}`;
+        reason = `Hostname: ${hostname} is not cert's CN: ${cn}`;
```

Comment:
The removal of the dot misses the, ah, point of unfqdn(), see the comment on line 209.  Including the FQDN in the error message is intentional and should not be changed without good reason.

Reply:
Ah right, got it, a comment here would be good then because that is most certainly *not* obvious.

Reply:
Maybe wrapping it in quotation marks might help it be mildly more obvious that it's not a typo?

```suggestion
        reason = `Host: "${hostname}." is not cert's CN: ${cn}`;
```